packages/amplify_storage_s3/ios/Classes/FlutterDownloadFileRequest.swift

```diff
@@ -26,15 +26,18 @@ struct FlutterDownloadFileRequest {
         self.options = setOptions(request: request)
     }
     
-    static func isValid(request: Dictionary<String, AnyObject>) -> Bool {
-        var valid: Bool = true;
+    static func validate(request: Dictionary<String, AnyObject>) throws {
+        let validationErrorMessage = "Download request malformed."
```

Comment:
It should be the API name `DownloadFile` as we may have `DownloadData` in the future. Same in Android code.

Reply:
Fixed.